product_listprice_list_view/models/product_template.py

```diff
@@ -38,11 +38,6 @@ class ProductTemplate(models.Model):
         string='Stock Lead Time',
         compute='_get_stock_location',
     )
-    net_price_cny = fields.Float(
-        string='Sale RMB',
-        compute='_get_net_price_cny',
-        digits=dp.get_precision('Product Price'),
-    )
```

Comment:
This PR seems incomplete by itself as it removes the field `net_price_cny` from `product_listprice_list_view` while the field is not added in `product_offer`, and you are still using the field in view records in this PR.